lang/zh-CN.json

```diff
@@ -41,9 +41,8 @@
   "Audio Player": "音频播放器",
   "Video Player": "视频播放器",
   "Replay": "重播",
-  "Progress Bar": "进度小节",
+  "Progress Bar": "进度条",
   "Volume Level": "音量",
-  "Close Modal Dialog": "关闭弹窗",
```

Comment:
why remove this item?

Reply:
It's duplicate in both [lang/zh-CN.json](https://github.com/videojs/video.js/blob/master/lang/zh-CN.json) and [lang/zh-TW.json](https://github.com/videojs/video.js/blob/master/lang/zh-TW.json)

search the key `Close Modal Dialog` and You will get two result.

Reply:
oh, didn't notice, cool!